<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оптимизация</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ввод данных</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>231</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть готовый файл задачи</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtOpenProblemFile">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>541</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenProblemFile">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpen">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать новую задачу</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Файл с матрицей</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtOpenMatrix">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>541</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenMatrix">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>130</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать файл</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Файл с вектором ограничений</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtOpenConstraints">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>541</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenConstraints">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>180</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать файл</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>261</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Файл с вектором целевой функции</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtOpenObjective">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>541</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenObjective">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать файл</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Целочисленные переменные</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>421</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Файл с булевым вектором целочисленности переменных</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtOpenInteger">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>541</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenInteger">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>300</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCreate">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Выбор задачи</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать оптимизатор</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboSolver">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>501</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSolve">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Решить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Значение целевой функции</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtValue">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>461</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Отправка на NEOS</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboNEOS">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>501</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать оптимизатор</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>401</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать файл для экспорта матриц в текст на языке LP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtExportLP">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>541</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExportLP">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>180</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать файл</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;E-mail&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtEmail">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>611</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSend">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Отправить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер запроса</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNumber">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>270</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>270</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>231</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить найденные решения</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveProblemFile">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>320</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать файл</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSaveProblemFile">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>541</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSave">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
